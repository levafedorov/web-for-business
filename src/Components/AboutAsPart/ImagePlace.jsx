import React from 'react';




export default function ImagePlace(props){
     


  
    return <section className="image-place">
                
    </section>
}