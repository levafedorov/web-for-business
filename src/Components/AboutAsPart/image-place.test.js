import React from 'react';
import {shallow} from 'enzyme';
import ImagePlace from './ImagePlace';

describe("ImagePlace component", ()=>{

    test("Should ImagePlace render", ()=>{
       const component = shallow(<ImagePlace/>);
       expect(component).toMatchSnapshot();    
    });

});