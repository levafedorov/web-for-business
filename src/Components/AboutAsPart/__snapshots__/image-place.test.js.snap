// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagePlace component Should ImagePlace render 1`] = `
<section
  className="image-place"
/>
`;
